docs/build/iota-sdk/1.0/docs/how-tos/nfts/mint-nft.mdx

```diff
@@ -108,19 +108,18 @@ https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how
 <div className={'hide-code-block-extras'}>
 
 ```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/mint_nft.ts#L55-L62
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/how_tos/nfts/mint_nft.ts#L61-L74
 ```
 
 </div>
 
-4. Prepare the transaction that will mint the NFTs using the [`Account.prepareMintNfts()`](../../references/nodejs/classes/Account.md#preparemintnfts)
-   function and send the prepared transaction using the [`Account.send()`](../../references/nodejs/classes/Account.md#send)
+4. Create the transaction that will mint the NFTs using the [`Account.mintNfts()`](../../references/nodejs/classes/Account.md#mintnfts) <!-- TODO Update ref link --> 
```

Comment:
```suggestion
4. Create the transaction that will mint the NFTs using the [`Account.mintNfts()`](../../references/nodejs/classes/Account.md#mintnfts) 
```

Reply:
We should leave them until I fix the refs